index.html

```diff
@@ -1496,9 +1495,9 @@ <h2 id="BaseAudioContext">
                     <span role="img" aria-label="False">✘</span>
                   </td>
                   <td class="prmDesc">
-                    A sequence of cosine parameters. See its <a href=
-                    "#widl-PeriodicWaveOptions-real">constructor
-                    counterpart</a> for a more detailed description.
+                    A sequence of cosine parameters. See its <a data-link-for=
+                    "PeriodicWaveOptions">constructor counterpart</a> for a
```

Comment:
This link doesn't work.

I'm ok if you want to fix this broken link in a later PR.